sound/soc/sof/intel/Kconfig

```diff
@@ -319,6 +319,7 @@ config SND_SOC_SOF_HDA_COMMON_HDMI_CODEC
 	bool "SOF common HDA HDMI codec driver"
 	depends on SND_SOC_SOF_HDA_LINK
 	depends on SND_HDA_CODEC_HDMI
+	default y if SND_HDA_CODEC_HDMI
```

Comment:
default SND_HDA_CODEC_HDMI ?

Reply:
@plbossart ok, potentially better

Reply:
@plbossart ack, seems to work. fixed in the update I just pushed